src/csm_bot/events.py

```diff
@@ -171,6 +171,17 @@ async def public_release(self, event: Event):
         template: callable = EVENT_MESSAGES.get(event.event)
         return template() + self.footer(event)
 
+    async def has_active_keys(self, node_operator_id: int) -> bool:
+        """Check if a node operator has active (non-withdrawn) keys."""
+        try:
+            node_operator = await self.csm.functions.getNodeOperator(node_operator_id).call()
+            # Active keys = deposited keys that haven't been withdrawn
+            active_keys = node_operator.totalDepositedKeys - node_operator.totalWithdrawnKeys
+            return active_keys > 0
+        except Exception:
```

Comment:
pls cover only rpc call with exception block and log a warning if there are errors appeared